Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -2162,6 +2168,25 @@ def get_store_data(service):
         yield store.data.query(**query)
 
 
+class MessagesHandler:
+
+    service = None
+
+    @staticmethod
+    def get_messages_data(notable_id):
+        if MessagesHandler.service is not None:
```

Comment:
this way you can decrease the area under the indent (for the rest of the function)
```suggestion
        if MessagesHandler.service is None:
            return []
```

Reply:
Fixed